_includes/getting-started.html

```diff
@@ -1,11 +1,7 @@
 <div class="container" style="width: 100%; text-align: left; margin-top: 30px;">
 
-  <h1 class="getting-started-heading">Getting Started</h1>
-    <h2 style>with Meshery</h2>
-      <p style="font-size: 1.15rem;">
-        Choose a platform and see how easy it is to start managing service meshes in a single command.
-      </p>
-    </h1>
+  <h1 class="getting-started-heading">Getting Started with Meshery</h1>
```

Comment:
The style in the Getting Started heading was intentional. Can you change it back to the previous format?

Reply:
sure, no problem.

Reply:
done as requested. @debo19